rts/at/webservice/pom.xml

```diff
@@ -50,12 +50,6 @@
       <artifactId>narayana-jta</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>asm</groupId>
-      <artifactId>asm-all</artifactId>
-      <version>${version.asm}</version>
-      <scope>test</scope>
-    </dependency>
```

Comment:
I'll add the comment here but this message is valid for all `asm` removals: is this dependency removal related to JBTM-1801?

Reply:
I just noticed you used a different commit to remove `asm`, which is alright for me.